internal/commands/app/create.go

```diff
@@ -354,11 +342,102 @@ func (cmd *CommandCreate) Handler(profile *user.Profile, ui terminal.UI, clients
 	return nil
 }
 
+func (cmd CommandCreate) handleCreateTemplateApp(
+	profile *user.Profile,
+	ui terminal.UI,
+	clients cli.Clients,
+	groupID string,
+	rootDir string,
+	dsClusters []dataSourceCluster,
+	dsDatalakes []dataSourceDatalake,
+) error {
+	if cmd.inputs.DryRun {
+		logs := make([]terminal.Log, 0, len(dsClusters)+len(dsDatalakes)+2)
+		logs = append(logs, terminal.NewTextLog(fmt.Sprintf(
+			"A Realm app would be created at %s using the '%s' template",
+			rootDir,
+			cmd.inputs.Template,
+		)))
+
+		for _, cluster := range dsClusters {
+			logs = append(logs, terminal.NewTextLog("The cluster '%s' would be linked as data source '%s'", cluster.Config.ClusterName, cluster.Name))
+		}
+		for _, datalake := range dsDatalakes {
+			logs = append(logs, terminal.NewTextLog("The data lake '%s' would be linked as data source '%s'", datalake.Config.DatalakeName, datalake.Name))
+		}
+		logs = append(logs, terminal.NewFollowupLog("To create this app run", cmd.display(true)))
+		ui.Print(logs...)
+		return nil
+	}
+
+	if len(dsClusters) != 1 {
+		return errors.New("could not send app creation request due to unexpected number of clusters")
```

Comment:
the error message here will ultimately be prefixed with `[command.display] failed: `... can we change the error message here to `"must specify a cluster when creating app from template"` or something along those lines?